Libs/Widgets/ctkErrorLogWidget.cpp

```diff
@@ -186,6 +189,35 @@ void ctkErrorLogWidget::setColumnHidden(int columnId, bool hidden) const
   d->ErrorLogTableView->setColumnHidden(columnId, hidden);
 }
 
+// --------------------------------------------------------------------------
+void ctkErrorLogWidget::setLayoutOrientation(Qt::Orientation orientation)
+{
+  Q_D(ctkErrorLogWidget);
+  if (d->LayoutOrientation == orientation)
+    {
+    return;
+    }
+
+  d->ctkErrorLogGridLayout->removeWidget(d->ErrorLogDescription);
+  if (orientation == Qt::Orientation::Horizontal)
+    {
+    d->ctkErrorLogGridLayout->addWidget(d->ErrorLogDescription, 0, 1, -1, 1);
+    }
+  else
+    {
+    d->ctkErrorLogGridLayout->addWidget(d->ErrorLogDescription, 2, 0, 1, 1);
+    }
```

Comment:
For sake of being explicit and help with reading the code:

```suggestion
d->ctkErrorLogGridLayout->removeWidget(d->ErrorLogDescription);
int fromRow = (orientation == Qt::Orientation::Vertical ? 2 : 0);
int fromColumn = (orientation == Qt::Orientation::Vertical ? 0 : 1);
int rowSpan = (orientation == Qt::Orientation::Vertical ? 1 : -1); // Specifying -1 ensures the widget extend to the right edge
int columnSpan = (orientation == Qt::Orientation::Vertical ? 1 : 1);
d->ctkErrorLogGridLayout->addWidget(d->ErrorLogDescription, fromRow, fromColumn, rowSpan, columnSpan);
```